docs/guidance/build/hosting.md

```diff
@@ -1,6 +1,6 @@
 # Data files and APIs
 
-Different users have different needs when it comes to accessing OCDS data. 
+Different users have different needs when it comes to accessing OCDS data. It also must be possible to automate the download of all the OCDS data, in some form.
```

Comment:
See this bullet from https://ocds-standard-development-handbook.readthedocs.io/en/latest/standard/contributing.html

> Never use normative words on guidance pages. Use [non-normative synonyms](https://tools.ietf.org/html/draft-hansen-nonkeywords-non2119-04#page-3) instead.

```suggestion
Different users have different needs when it comes to accessing OCDS data. It also needs to be possible to automate the download of all the OCDS data, in some form.
```